docs/guidelines.md

```diff
@@ -1,2 +1,33 @@
-# Basic Guidelines to follow while integration
+# Guidelines to follow while integration
+This section captures the guidelines for broad use cases which relates to almost all of our Ultra partners’ applications.
 
+### General
+These general guidelines apply across all the pages within a partner’s application.
+
+- No external redirection in your app.
+- We do not tolerate obstruction of Flipkart’s property within Ultra.
+- Your app experience must have parity.
+
+### Component-wise
+Here are the instructions to follow for special components within Ultra:
+
+#### Entry
+- ***Login or Sign-up:*** Do not have your own login or sign-up flows. Disable all paths in your application that lead to any of these pages. Same pertains to the log out and account modification (password change, profile change, etc.) processes.
+- ***Homepage:*** Do not include promotions that belong to non-Flipkart properties and the advertisements from external parties on your homepage.
+
+#### Order Funnel
+- ***Product Discovery:*** There are no explicit restrictions around flows such as the Search, the Flyout menu and other navigation panels. But you must take care of UI elements that might confuse the user. For example, the flyout menu must not be identical to Flipkart’s flyout menu, etc.
+- ***Offers:*** At any point in the funnel, do not display the offers that customers cannot avail within Flipkart’s context. Even, we would not allow any instructional notes. For example, messages that show how to avail an offer in another platform.
```

Comment:
Please see if the sentence "Even, we would not allow any instructional notes" be reworded.

Reply:
Done. Please check the other commit of the same branch (OjesviPandita-Upd-Guidelines).